### ISSUE TITLE
Lighthouse Testing

### ISSUE THREAD

**Issue by irmakerdem** [ASands17/fitlit#94]
You must score as close to 100% as possible with the “Lighthouse Accessibility Audit”. Be prepared to explain any accessibility audits your application is failing.
